test/functions/deitz/named/test_named6-error.good

```diff
@@ -1 +1 @@
-test_named6-error.chpl:5: error: The named argument 'i' is used more than once in the same function call.
+test_named6-error.chpl:5: error: The named argument 'i' is used more than once in the same function call
```

Comment:
Is this being generated by a brief error now?  Shouldn't it be automatically generating an initial lowercase as a result?  We could even drop the "The" if desired to make it brief (though I suspect this form will read better in the long form, so am not necessarily advocating for it).

Reply:
The "brief" error messages in the compiler are inconsistent, so I am making the brief be "whatever the programmer wrote" and "detailed" be "start with uppercase". This error message is written with an uppercase in the code, so it's left with an uppercase in brief mode. I have nothing against correcting that, though.

Reply:
I think it'd be good if, at some point during this process, we were to make them consistent in this regard, as I don't think the current mix is due to intention or design, but just different people doing different things in different situations.  This PR need not take on that challenge, though.  I was just imagining that the compiler was forcing lowercase for brief and uppercase for long, but now I understand what you've done better.

It's an interesting question how to enforce this going forward though.  We could have the compiler warn/error if/when a brief error leads with uppercase, but is that too heavy-handed?  E.g., are there ever cases where the capitalization is due to something other than "I'm starting a sentence" like an acronym or a proper noun?  I suppose we could hope/expect not and deal with that case if/when we encounter one.